### ISSUE TITLE
Compilation error. "Type mismatch"

### ISSUE THREAD

**Issue by ThinkDigitalRepair** [sidlatau/flutter_email_sender#5]
After updating my flutter version to 
```
flutter --version
Flutter 0.10.1-pre.40 • channel master • https://github.com/flutter/flutter.git
Framework • revision 8e2ca93f52 (18 hours ago) • 2018-10-12 21:07:55 -0700
Engine • revision 50c2e69daf
Tools • Dart 2.1.0-dev.7.0.flutter-b99bcfd309
```
I'm getting the following error.
``` 
/Users/thinkdigital/development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_email_sender-
1.0.0/android/src/main/kotlin/com/sidlatau/flutteremailsender/FlutterEmailSenderPlugin.kt: (60, 66):
Type mismatch: inferred type is kotlin.collections.ArrayList<String>? /* = java.util.ArrayList<String>? */ 
but kotlin.collections.ArrayList<String> /* = java.util.ArrayList<String> */ was expected
```

This issue doesn't show up in 
Flutter 0.9.4 • channel beta

**Comment by sidlatau** [comment]
Thanks for registering an issue! I released new v1.0.1 plugin version that fixes the problem.

**Comment by ThinkDigitalRepair** [comment]
You're welcome!
